spanner/google/cloud/spanner_v1/streamed.py

```diff
@@ -48,13 +48,13 @@ def __init__(self, response_iterator, source=None):
         self._source = source       # Source snapshot
 
     @property
-    def rows(self):
+    def _rows(self):
```

Comment:
why is this method needed? Can we change everywhere this is referenced to directly reference `_processed_rows`